how-we-work/tracking-time.md

```diff
@@ -1,116 +1,114 @@
+overtime as needed:
 
-### Leave and overtime hours
+- if you have worked **less** than the expected hours, you should report time off;
+- if you have worked **more** than the expected hours, you should report overtime.
 
-In order to calculate the hours spent at the office, just calculate the start and end time of a
-working day, subtracting the lunch break time (or any other major break).
-This way you will know if you worked the correct number of hours or if you have
-to [mark leave or overtime hours.](https://github.com/nebulab/playbook/blob/master/how-we-work/tools.md#dipendenti-in-cloud)
+**NOTE:** You CANNOT "compensate" time off with overtime. For instance, you cannot take 30 minutes
+off today and stay 30 minutes more tomorrow _without_ reporting it, but it is perfectly acceptable
+to do it if you report 30 minutes of time off today and 30 minutes of overtime tomorrow.
 
-An handy tool you can use to count the number of worked hours is this
-[time sheet calculator](http://www.miraclesalad.com/webtools/timesheet.php).
+## Hours of work
 
-One very important rule of conduct is to always mark the hours of leave and overtime day by day. In
-fact, if you have to take time off during a working day, due to an emergency or any other reason,
-that leave of absence, short as it may be, you need to track it so that you remember to make up for
-the lost time over the coming days. Tracking your hours constantly and transparently helps you be
-fair towards clients and colleagues.
+These are the hours you actually spend on work tasks and are tracked via Harvest. Keep in mind that
+Harvest is NOT a tool for controlling employees. We use it in order to be 100% transparent towards
+our clients and to improve our time management skills. If you are worried about it, reach out to
+your mentor and they will be able to answer all of your questions!
 
-Examples:
+Our Harvest account is setup with various projects and each project has several subtasks. Generally
+speaking, each client project gets its own project in Harvest and has tasks for Software
+Development, Project Management and so on. All of these tasks are **billable**, meaning any time you
+track will show up in the client's invoice at the end of the month. This means we need to be extra
+careful not to bill any time we don't spend working for a client.
 
-- **NO**: "I take half an hour off today, I will make up for it tomorrow" without marking anything
-  on Dipendenti in Cloud;
-- **YES**: "I take half an hour off today, I will make up for it tomorrow" marking a half a hour of
-  time off today and half an hour of overtime the next day.
+Unfortunately, the line between billable and non-billable time is not always clear and, as each
+project is unique, there is no universal rule we can set. We trust you to make the right decisions
+in accordance with [our values](https://github.com/nebulab/playbook/blob/master/about-us/our-values.md)
+and offer some helpful hints below.
```

Comment:
`and *we* offer`, slightly less ambiguous, although it's clear taking the context into account